Tarea clase 5 ejercicio 2

Mariano19 opened the pull request:
Ejercicio Clase 5 Tarea 2 resuelto.

Mariano19 committed:
Tarea clase 5 ejercicio 2

Ejercicio resuelto.

fsodano commented on tareas/clase-5/tarea2-mio.html:
```diff
@@ -0,0 +1,34 @@
+    <form>
+       
+        <label for="primer-nombre">Primer Nombre:</label>
+        <input type="text" id="primer-nombre" placeholder="Ej.: Mariano"/>
```
bien los nombres! `kebab-case` 👍

fsodano commented on tareas/clase-5/tarea2-mio.html:
```diff
@@ -0,0 +1,34 @@
+    </form>
+    <script type="text/javascript" src="tarea2-mio.js"></script>
+</body>
+</html>
```
Necesitás una nueva línea al final del archivo

fsodano commented on tareas/clase-5/tarea2-mio.js:
```diff
@@ -0,0 +1,29 @@
+let resumen = document.querySelector('#datos-usuario')
+resumen.onclick = function(){
+    const nombreUsuario = document.querySelector('#primer-nombre').value
+    const nombreUsuario2 = document.querySelector('#segundo-nombre').value
```
```javascript
    const primerNombre = document.querySelector('#primer-nombre').value
    const segundoNombre = document.querySelector('#segundo-nombre').value
```

fsodano commented on tareas/clase-5/tarea2-mio.js:
```diff
@@ -0,0 +1,29 @@
+    const nuevoParrafo = document.createElement('p')
+    const nuevoTexto = document.createTextNode(nombreUsuario + nombreUsuario2 + apellidoUsuario + edadUsuario)
+    nuevoParrafo.appendChild(nuevoTexto)
+    body.appendChild(nuevoParrafo)
```
Todo esto estaría piola ponerlo en una función separada.

Mariano19 replied:
Estoy intentando hacerlo en dos funciones pero por el tema del scope tengo que sacar las variables afuera. Esta bien?

Mariano19 committed:
Correccion tarea 2

Agrega una funcion mas para tener un codigo mas limpio y ordenado

Mariano19 committed:
ejercicio 2

Mas correcciones, cambio de nombre variables y arreglo linea al final del archivo

Mariano19 committed:
Tarea clase 5 ejercicio 4

Resuelvo ejercicio utilizando funciones
